shared/src/business/utilities/pdfGenerator/documentTemplates/StandingPretrialNotice.jsx

```diff
@@ -0,0 +1,119 @@
+const React = require('react');
+const { DocketHeader } = require('../components/DocketHeader.jsx');
+const { PrimaryHeader } = require('../components/PrimaryHeader.jsx');
+
+export const StandingPretrialNotice = ({ options, trialInfo }) => {
```

Comment:
This component will have to be added to `documentGenerators.js`, `reactTemplateGenerators.js`, and added to the `getDocumentGenerators()` method in `web-api/applicationContext.js` if you're wanting this to be a generated document. 

CC: @matthopson - I miss anything?

Reply:
I messed up my local staging branch and so this file was erroneously included. I deleted it to match staging.